<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>700</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Snake Identifier</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QGroupBox" name="inputGroupBox">
            <property name="title">
              <string>Входное изображение</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
                <widget class="QLabel" name="originalImageLabel">
                  <property name="minimumSize">
                    <size>
                      <width>300</width>
                      <height>300</height>
                    </size>
                  </property>
                  <property name="frameShape">
                    <enum>QFrame::Box</enum>
                  </property>
                  <property name="text">
                    <string>Оригинальное изображение</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignCenter</set>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="processedImageLabel">
                  <property name="minimumSize">
                    <size>
                      <width>300</width>
                      <height>300</height>
                    </size>
                  </property>
                  <property name="frameShape">
                    <enum>QFrame::Box</enum>
                  </property>
                  <property name="text">
                    <string>Обработанное изображение</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignCenter</set>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="QGroupBox" name="controlsGroupBox">
            <property name="title">
              <string>Управление</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
                <widget class="QPushButton" name="selectImageButton">
                  <property name="text">
                    <string>Выбрать фото</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QPushButton" name="processButton">
                  <property name="text">
                    <string>Обработать и найти</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="label">
                  <property name="text">
                    <string>Уровень обработки:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QSlider" name="preprocessingLevel">
                  <property name="minimum">
                    <number>1</number>
                  </property>
                  <property name="maximum">
                    <number>5</number>
                  </property>
                  <property name="value">
                    <number>3</number>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </widget>
              </item>
              <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="showStepsCheckBox">
                  <property name="text">
                    <string>Показать этапы обработки</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="QGroupBox" name="resultGroupBox">
            <property name="title">
              <string>Результаты</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
                <widget class="QLabel" name="resultLabel">
                  <property name="font">
                    <font>
                      <pointsize>12</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                    </font>
                  </property>
                  <property name="text">
                    <string>Результат появится здесь</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignCenter</set>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="matchedImageLabel">
                  <property name="minimumSize">
                    <size>
                      <width>300</width>
                      <height>300</height>
                    </size>
                  </property>
                  <property name="frameShape">
                    <enum>QFrame::Box</enum>
                  </property>
                  <property name="text">
                    <string>Совпадение из базы</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignCenter</set>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QGroupBox" name="saveGroupBox">
                  <property name="title">
                    <string>Сохранение в базу</string>
                  </property>
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                      <widget class="QLabel" name="label_2">
                        <property name="text">
                          <string>Имя змеи:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QLineEdit" name="snakeNameEdit"/>
                    </item>
                    <item>
                      <widget class="QPushButton" name="saveToDbButton">
                        <property name="text">
                          <string>Сохранить в БД</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="QProgressBar" name="progressBar">
            <property name="value">
              <number>0</number>
            </property>
            <property name="textVisible">
              <bool>true</bool>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>